overlord/snapstate/export_test.go

```diff
@@ -68,7 +68,7 @@ func (m *SnapManager) AddForeignTaskHandlers(tracker ForeignTaskTracker) {
 
 func MockReadInfo(mock func(name string, si *snap.SideInfo) (*snap.Info, error)) (restore func()) {
 	readInfo = mock
-	return func() { readInfo = snap.ReadInfo }
+	return func() { readInfo = readInfoImpl }
```

Comment:
With the usual pattern, we don't have to fix this every time we change readInfo:
```
old := readInfo
readInfo = mock
return func() { readInfo = old }
```